package.json

```diff
@@ -18,8 +18,8 @@
                         "type": "string"
                     },
                     "default": {
-                        "javascript": "node ${file}",
-                        "typescript": "tsx ${file}"
+                        "javascript": "node ${relativeFile}",
```

Comment:
Will its still work for files outside of workspace and when no workspace is opened? Default should be optimal for all cases

Reply:
will use absoluteFilePath